Source/UI/Screens/Identifier/IdentifierViewController.swift

```diff
@@ -21,6 +21,10 @@ class IdentifierViewController: IdentityUIViewController {
 
     @IBOutlet var whastThisButton: UIButton! {
         didSet {
+            guard !configuration.disableWhatsThisButton else {
+                whastThisButton.setTitle("", for: .normal)
```

Comment:
This button caused a rejection of our apps b/c the privacy page leads to a page with subscription purchase possibility.

Reply:
Why not hide it with `whastThisButton.isHidden = true` instead?

Reply:
Fair point, let me fix that